projects/storefrontapp-e2e-cypress/cypress/helpers/close-account.ts

```diff
@@ -67,9 +59,6 @@ export function closeAccountTest() {
     registerAndLogin();
```

Comment:
need to remove this assertion because we don't need to test that, the login will happen in the before as mentioned in my previous comment and also remove the cy.visit('/') in line 11

The above statement applies to close, email and password e2e test